Shastak/BulkThumbnailCreator/Extras/BTCLogger.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.IO;
+using BulkThumbnailCreator.Modules;
+
+namespace BulkThumbnailCreator.Extras
+{
+    public static class BTCLogger
+    {
+        private static string LogsFilename { get; set; }
+
+        public static void WriteLocalLogs(string message)
+        {
+            CreateLogsFileForToday();
+
+            using (StreamWriter sw = new StreamWriter(LogsFilename, true))
+            {
+                string now = DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss.fffff");
```

Comment:
Better to move date format to config or to separate class for string constants

Reply:
fixed by adding Constants.cs class